Devscord.DiscordFramework/WorkflowBuilder.cs

```diff
@@ -46,74 +48,91 @@ public WorkflowBuilder SetMessageHandler(Func<SocketMessage, Task> action)
 
         public WorkflowBuilder SetDefaultMiddlewares()
         {
+            Log.Debug("Default middlewares added");
```

Comment:
to nie powinno być po dodaniu? tak gdyby jakimś cudem coś się stało z AddMiddleware